### ISSUE TITLE
Открыта новая тема СВОДД 55

### ISSUE THREAD

**Issue by iprst** [terratensor/svodd#300]
ПРОДОЛЖЕНИЕ ОБСУЖДЕНИЯ СВОДД 55
[https://фкт-алтай.рф/qa/question/view-7615](https://xn----8sba0bbi0cdm.xn--p1ai/qa/question/view-7615)

![image](https://github.com/user-attachments/assets/2fdc5ef1-1ef7-4621-940e-c44a5e43bfb8)

Статистика подключена, телеграм подключен.

![image](https://github.com/user-attachments/assets/a82ba2a5-42c6-4f5a-93d9-6c4e3372ff1d)

![image](https://github.com/user-attachments/assets/9992be88-eedd-4eab-ad11-4c890af7805c)

Системы работают штатно.
Полёт нормальный.

**Comment by audetv** [comment]
Да, все штатно работает, после обновления не сломалось.

**Comment by iprst** [comment]
Добавил объявление про обновления сайта. Что-то телеграм странно пожал гифку, ранее такого не наблюдал. Если есть возможность, её лучше заменить на оригинал по ссылке в сообщении, а то сейчас выглядит странно.

**Comment by audetv** [comment]
Не знаю, не дает ничего сделать с препросмотром, можно переместить предпросмотр вверх, вниз. И больше ничего, уменьшить увеличить нельзя . наверное потому что это ссылка в сообщении, это не прикрепленный к сообщению файл. Видимо в этом случае телеграм сам все решает. Можно заменить текст ссылки на новый текст. Попробовал добавлять параметр к ссылке типа gif?v=1 но это ничего не дало. Можно попробовать заменить файл по ссылке. Или видео разместить, ссылку поменять я могу.

**Comment by iprst** [comment]
Понял. В таком случае лучше просто подменить гифку на подогранный вариант
https://cont.ws/uploads/pic/2024/7/searchmark.svodd.gif

**Comment by audetv** [comment]
Поменял ссылку.

**Comment by iprst** [comment]
Отлично. Всё равно жмёт, но теперь хотя бы всё видно. Пусть так. Судя по поведению телега жмёт не в геометрические размеры, а по размеру файла.  По таблицам Брадиса, блин.

**Comment by audetv** [comment]
Удивлен, что это сообщение об обновлении получило уже столько просмотров. 
Т.е. как будто на него смотрят, а рядом выше ниже не смотрят сообщения.

**Comment by iprst** [comment]
Да, точно. Замечаю такое поведение иногда. Кажется, если какое-то сообщение повисит некоторое время как самое новое, при удачном стечении обстоятельств оно наберёт какой-то перевес, который затем суммируется, если до и после него тему смотрят какие-то другие пользователи. Но потом те, кто посмотрел только эту запись, могут зайти и посмотреть тему рядом, и разница нивелируется. 

Можно сделать ход конём и добавить к нему в телеге тег Пякин с решёткой. Произойдёт _набегание_.

**Comment by audetv** [comment]
> Можно сделать ход конём и добавить к нему в телеге тег Пякин с решёткой. Произойдёт _набегание_.

Да, потому что Пякин и Вопрос Ответ. Думаю, что ваша гипотеза верна, но обойдёмся без _набеганий_ =)
